subprojects/libero-core/src/main/groovy/com/connexta/libero/Libero.groovy

```diff
@@ -50,34 +52,28 @@ class Libero {
         finalizeConfig(config)
 
         Date date = new Date()
```

Comment:
If this version of Groovy is based on Java 8, we could use `Instant.now().toString()` instead of the old and ugly `Date` class and `SimpleDateFormat`.